README.md

```diff
@@ -85,7 +88,7 @@ Add this to your CDK stack:
 ```typescript
 import { DatadogLambda } from "datadog-cdk-constructs-v2";
 
-const datadog = new DatadogLambda(this, "Datadog", {
+const datadogLambda = new DatadogLambda(this, "datadogLambda", {
```

Comment:
`datadogLambda` is the preferred variable name now because we will soon add `datadogSfn` for Ste pFunctions. Using `datadog` will cause confusion.